infrastructure/batch/roles.tf

```diff
@@ -22,60 +23,59 @@ resource "aws_iam_role_policy_attachment" "batch_service_role" {
   policy_arn = "arn:aws:iam::aws:policy/service-role/AWSBatchServiceRole"
 }
 
-resource "aws_iam_role" "ecs_task_execution_role" {
+resource "aws_iam_role" "ecs_task_role" {
```

Comment:
can you change these names back as well, or let me know if it is a required change